Nikki/Features/Onboarding/QA.md

```
@@ -80,6 +84,24 @@ last_verified_at: 2026-08-22
 
 </details>
 
+### **パスキー登録ボタンが出る**: 最終ステップに、生体認証を有効にする主ボタンの下に枠線の「パスキーを登録する」ボタンが出る (issue #84)
+
+<details><summary>動作確認スクショ</summary>
+
+**確認日: 2026-09-02**
+<img src="https://pub-7f3469dd3e2e445b9b8ec2d1381b5ea8.r2.dev/bannzai/nikki/20260902/11ba36bd-b2cb-4625-be88-230fd0689d70.jpg" width="320">
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)